src/mahoji/commands/mix.ts

```diff
@@ -171,6 +187,9 @@ export const mixCommand: OSBMahojiCommand = {
 		if (boosts.length > 0) {
 			str += `\n**Boosts:** ${boosts.join(', ')}`;
 		}
+		if (hasScroll) {
+			str += ' Your scroll of cleansing enables you to save 10% of your secondaries.';
+		}
```

Comment:
Preferably just put this in the boosts array. We're moving away from the `str += ` because its hard to keep track of formatting them.